assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -238,6 +266,28 @@ function SizePositionPanel({
           </Button>
         </Row>
       )}
+      {/** X/Y */}
+      <Row expand>
+        <BoxedNumeric
+          suffix={_x('X', 'Position on X axis', 'web-stories')}
+          value={x}
+          min={MIN_MAX.X.MIN - width}
```

Comment:
Hmm, just realized that for the limits to actually make sense, it should also consider rotation when subtracting width/height...

Reply:
Otherwise this would be allowed for example:
<img width="894" alt="Screenshot 2020-08-24 at 12 00 10" src="https://user-images.githubusercontent.com/3294597/91025022-10d17000-e5f9-11ea-924a-41a3011d9bfd.png">

Thoughts?

Reply:
True. While X&Y coords are not so easy to operate when rotation is involved - you need to do some math before you put something in the input, but when using a keyboard, the min/max limit would be nice.

BTW: Checked how Figma does this (different from us, they update the coords):
Box at 0x0
![image](https://user-images.githubusercontent.com/21036224/91169602-edc9bd80-e6d7-11ea-8bb5-4e64b693b1c5.png)

Coords are updated when rotation is applied
![image](https://user-images.githubusercontent.com/21036224/91169628-f8845280-e6d7-11ea-9aaf-7bb16ee678d0.png)

Move to 0x0 (not as some would like and not as we do it)
![image](https://user-images.githubusercontent.com/21036224/91169673-06d26e80-e6d8-11ea-9e58-dbc00966aad8.png)

Reply:
Yeah, there was this little tiny voice and the back of my head screaming this over and over but I managed to suppress it all this time. I've addressed it now in fe8118780.

Reply:
> BTW: Checked how Figma does this (different from us, they update the coords)

Yeah, that's left as an exercise for future improvement. It makes a lot of sense, that the input for x is always the left-most edge of an element, but it's not easy.

In this rewrite I am actually getting the information necessary, so we could simply translate it back and forth in the input only (and still use a single anchor corner internally), but it might come with other errors.